_positions/visual-designer.md

```diff
@@ -1,40 +1,29 @@
 ---
 title: Visual Designer
-permalink: who-we-are-hiring/positions/visual-designer/
+permalink: roles-and-teams/positions/visual-designer/
```

Comment:
Good call, now that the roles have been moved to `roles-and-teams`. Making an issue to note that these should change across the board.